app/screens/home/search/search.tsx

```diff
@@ -241,9 +243,20 @@ const SearchScreen = ({teamId, teams, crossTeamSearchEnabled}: Props) => {
         setResultsLoading(false);
     }, [lastSearchedValue, searchTeamId, serverUrl]);
 
+    const removeChannelAndUserFiltersFromString = (str: string) => {
+        return str.replace(CHANNEL_AND_USER_FILTERS_REGEX, '').trim();
+    };
```

Comment:
What I meant from my previous comment, is that you can move the whole function out. It doesn't depends on anything inside the component.